### PR TITLE
improved README

### PR DESCRIPTION
**Pull request by hritikgupta**
Signed-off-by: hritikgupta <hritikgupta9@gmail.com>
- Enhanced language
- Fixed grammatical and typographical errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
-
-So generally developer write there own vulnerable applications but that causes productivity loss and also many times rework is done. This Project VulnerableApp is build keeping these factors in mind so this project is scalable, extensible, easiers to integrate and easier to learn.
+As Web Applications are becoming popular these days, there comes a dire need to secure them. Although there are several Vulnerability Scanning Tools, however while developing these tools, developers need to test them. Moreover, they also need to know how well is the Vulnerability Scanning tool performing. As of now, there are little or no such vulnerable applications existing for testing such tools. There are Deliberately Vulnerable Applications existing in the market but they are not written with such an intent and hence lag extensibility, e.g. adding new vulnerabilities is quite difficult. Hence, the developers resort to writing their own vulnerable applications, which usually causes producitivity loss and the pain to rework.
```
Comment (preetkaran20) [inline README.md:12]:
```suggestion
As Web Applications are becoming popular these days, there comes a dire need to secure them. Although there are several Vulnerability Scanning Tools, however while developing these tools, developers need to test them. Moreover, they also need to know how well is the Vulnerability Scanning tool performing. As of now, there are little or no such vulnerable applications existing for testing such tools. There are Deliberately Vulnerable Applications existing in the market but they are not written with such an intent and hence lag extensibility, e.g. adding new vulnerabilities is quite difficult. Hence, the developers resort to writing their own vulnerable applications, which usually causes productivity loss and the pain to rework.
```

File: README.md [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
 
-As solving the above issue requires addition of various vulnerabilities, hence it becomes a very good platform to learn various security vulnerabilities. 
+**VulnerableApp** is built keeping these factors in mind. This project is scalable, extensible, easier to integrate and easier to learn.
+As solving the above issue requires addition of various vulnerabilities, hence it becomes a very good platform to learn various security vulnerabilities.
 
 ### Future Goal
-As going further this application might becomes a database for vulnerabilities hence in future it can be used for hosting CTF's and also can be come a compliance for Vulnerability Scanning tools.
 
-## How to use this tool ##
+Going further, this application might becomes a database for vulnerabilities. Hence, in future, it can be used for hosting CTFs and can also become a compliance for Vulnerability Scanning tools.
```
Comment (preetkaran20) [inline README.md:19]: I am not sure if we can change compliance to benchmark or make it compliance/benchmark

**Comment by preetkaran20** [comment]
@hritikgupta  thanks a lot for improving Readme. 

Please also update https://github.com/SasanLabs/VulnerableApp/blob/master/docs/index.md file. This file is similar to Readme and used in documentation. Documentation Url: https://sasanlabs.github.io/VulnerableApp/

thanks,
Karan

**Comment by preetkaran20** [comment]
@hemantgs  please review.

thanks,
Karan

File: README.md (hunk 2 above)
Comment (hritikgupta) [reply README.md:19]: Going ahead with the latter sounds better to me

File: README.md [hunk 3]
```diff
@@ -1,34 +1,41 @@
...
 ### Future Goal
-As going further this application might becomes a database for vulnerabilities hence in future it can be used for hosting CTF's and also can be come a compliance for Vulnerability Scanning tools.
 
-## How to use this tool ##
-[How to use guide](https://github.com/SasanLabs/VulnerableApp/blob/master/HOW-TO-USE.md)
+Going further, this application might becomes a database for vulnerabilities. Hence, in future, it can be used for hosting CTFs and can also become a compliance/benchmark for Vulnerability Scanning tools.
+
+## How to use this tool
+
+[How to use guide](https://github.com/SasanLabs/VulnerableApp/blob/master/docs/HOW-TO-USE.md)
```
Comment (preetkaran20) [inline README.md:23]: Great!!!

**Review by preetkaran20 — APPROVED** [review]
LGTM. Thanks!!

**Comment by preetkaran20** [comment]
@hritikgupta  Shall i merge this PR ?

**Comment by hritikgupta** [comment]
@preetkaran20 sure!

**Pull request merged**